webapp/pom.xml

```diff
@@ -62,6 +62,46 @@
       <artifactId>log4j-web</artifactId>
     </dependency>
   </dependencies>
+  
+	<build>
+		<defaultGoal>package</defaultGoal>
+		<plugins>
+			<!--  unpack META-INF service from core.jar to include it in WEB-INF/classes/META-INF, to be found in WebSphere -->
```

Comment:
I don't think this is just websphere. I think it works locally (with Eclipse) because it adds the projects to the classpath. I've had similar issues in the past. Sometimes less is more, _copy META-INF/services from the core module so it can be found by application servers._